jsk_perception/CMakeLists.txt

```diff
@@ -246,23 +246,25 @@ add_dependencies(hough_lines              ${PROJECT_NAME}_gencfg ${PROJECT_NAME}
 add_dependencies(rectangle_detector       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
 add_dependencies(color_histogram_sliding_matcher       ${PROJECT_NAME}_gencfg ${PROJECT_NAME}_gencpp)
 
+# FIXME: actually, roseus executable is usable only on Linux
+if(GNU)
```

Comment:
> humm rospack_LIBRARIES depends on libpython, this not happens in linux

Is it strange? But it seems normal.
https://github.com/ros/rospack/blob/jade-devel/CMakeLists.txt#L32
```
% ldd /opt/ros/indigo/bin/rospack | grep python
	libpython2.7.so.1.0 => /usr/lib/x86_64-linux-gnu/libpython2.7.so.1.0 (0x00007fd3a0b5d000)

% lsb_release -r
Release:	14.04
wkentaro at anaconda in ~
% lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 14.04.4 LTS
Release:	14.04
Codename:	trusty
```